xml/libvirt_migrating_vms.xml

```diff
@@ -0,0 +1,605 @@
+        <listitem>
+<screen>&prompt.user;virsh migrate 37 qemu+ssh://&exampleuser_plain;@&wsIname;/system</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Transient live migration with default parameters</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate --live opensuse131 qemu+ssh://&exampleuser_plain;@&wsIname;/system</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Persistent live migration; delete VM definition on source</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate --live --persistent --undefinesource 37 \
+qemu+tls://&exampleuser_plain;@&wsIname;/system</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Non-live migration using port 49152</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate opensuse131 qemu+ssh://&exampleuser_plain;@&wsIname;/system \
+--migrateuri tcp://@&wsIname;:49152</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Offline migration transferring all used storage</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate --offline --persistent --copy-storage-all \
+opensuse156 qemu+ssh://&exampleuser_plain;@&wsIname;/system</screen>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+
+    <note>
+      <title>Transient compared to persistent migrations</title>
+      <para>
+        By default <command>virsh migrate</command> creates a temporary
+        (transient) copy of the &vmguest; on the target host. A shut down
+        version of the original guest description remains on the source host. A
+        transient copy is deleted from the server after it is shut down.
+      </para>
+      <para>
+        To create a permanent copy of a guest on the target host, use the switch
+        <option>--persistent</option>. A shut down version of the original guest
+        description remains on the source host, too. Use the option
+        <option>--undefinesource</option> together with
+        <option>--persistent</option> for a <quote>real</quote> move where a
+        permanent copy is created on the target host and the version on the
+        source host is deleted.
+      </para>
+      <para>
+        It is not recommended to use <option>--undefinesource</option> without
+        the <option>--persistent</option> option, since this results in the loss
+        of both &vmguest; definitions when the guest is shut down on the target
+        host.
+      </para>
+    </note>
+  </sect1>
+<!-- Step by step example -->
+  <sect1 xml:id="sec-libvirt-migrate-stepbstep">
+    <title>Step-by-step example</title>
+
+    <sect2 xml:id="sec-migrate-stepbstep-export">
+      <title>Exporting the storage</title>
+      <para>
+        First you need to export the storage, to share the Guest image between
+        host. This can be done by an NFS server. In the following example we
```

Comment:
```suggestion
        First, you need to export the storage to share the guest image between
        hosts. This can be done by an NFS server. In the following example, we
```